src/javascript/app/base/header.js

```diff
@@ -656,9 +656,10 @@ const Header = (() => {
         const isOAuth2Enabled = AuthClient.isOAuth2Enabled();
 
         if (isOAuth2Enabled) {
-            const redirectCallbackUri = `${window.location.origin}/en/callback`;
+            const currentLanguage = Language.get();
```

Comment:
This should already have default fallback to `EN` in the function:
<img width="280" alt="Screenshot 2024-12-10 at 2 45 38 PM" src="https://github.com/user-attachments/assets/a1e565ef-02cc-4cad-8241-b77428b1d010">